manual_run/map_columns.py

```diff
@@ -0,0 +1,53 @@
+from typing import Dict, List
+
+from manual_run.variables import Variables
+
+
+class MapColumns:
```

Comment:
The pythonic way of doing this is to realize that any time you have a class where the class's only purpose is to provide static methods, you probably don't need a class and could have just defined the functions in this file without the class